radixdlt-java/src/main/java/com/radixdlt/client/atommodel/tokens/TokenDefinitionParticle.java

```diff
@@ -69,7 +69,7 @@ public TokenDefinitionParticle(
 		this.description = description;
 		this.granularity = granularity;
 		this.tokenPermissions = ImmutableMap.copyOf(tokenPermissions);
-		this.icon = icon;
+		this.iconUrl = iconUrl;
```

Comment:
should we perform any validation at all? Should we allow the empty string? null? a MASSIVE string? Should we validate that it is indeed a valid URL?

Reply:
The RadixCore software ensures that, if present, the URL is valid.  I don't think it makes much sense to duplicate that validation here, as if the application is accepting arbitrary input from users, it will likely want to do some validation there to ensure that inputs are acceptable.

Null is permitted.  Feel free to use as large a string as you can afford :)